src/client/marble-manager.js

```diff
@@ -125,9 +126,16 @@ const makeTextSprite = function(message, options = {}) {
 	let texture = new THREE.Texture(canvas);
 	texture.needsUpdate = true;
 
-	let spriteMaterial = new THREE.SpriteMaterial({ map: texture });
+	let spriteMaterial = new THREE.SpriteMaterial({ 
+		map: texture,
+		sizeAttenuation: false,
+		depthWrite: false,
+		depthTest: false,
+	});
+
 	let sprite = new THREE.Sprite(spriteMaterial);
-	sprite.scale.set(4, 1, 1.0);
+	sprite.scale.set(0.65, 0.25, 1.0);
+	sprite.renderOrder = options.drawOnTop || 1;
```

Comment:
Why bother setting the renderOrder if no custom one is defined? It's better to leave it at the default. I would suggest something more like `if (options.drawOnTop) sprite.renderOrder = options.drawOnTop;` (but with a better name, as mentioned in an earlier comment)